go/inst/instance_topology.go

```diff
@@ -1735,6 +1764,12 @@ Cleanup:
 	}
 	AuditOperation("take-master", instanceKey, fmt.Sprintf("took master: %+v", masterInstance.Key))
 
+	// Created this to enable a custom hook to be called after a TakeMaster success.
+	// This only runs if set to true in orchestrator.conf.json
+	if config.Config.PostTakeMasterProcessesEnabled == true {
+		TakeMasterHook(instance, masterInstance)
```

Comment:
in this function, `instance` is the old replica, now successor. `masterInstance` is the old master, now replica. But in `TakeMasterHook` you name them `master` and `replica`, respectively, which is the other way around. Please keep the naming consistent, or avoid calling them `master` and `replica`, and instead use `successor` and `demoted`.

Reply:
I've pushed the changes, please let me know if this is ok. Thanks for all the help and input 😄  👍